libraries/GCS_MAVLink/GCS.cpp

```diff
@@ -48,18 +48,15 @@ void GCS::send_to_active_channels(uint32_t msgid, const char *pkt)
         return;
     }
     for (uint8_t i=0; i<num_gcs(); i++) {
-        GCS_MAVLINK &c = chan(i);
-        if (!c.initialised) {
+        GCS_MAVLINK *c = chan(i);
```

Comment:
Not critical, but the diff could have been minimized in this area.
```suggestion
        GCS_MAVLINK &c = *chan(i);
```

Reply:
Done, thanks.